homeassistant/components/energyzero/sensor.py

```diff
@@ -200,3 +187,18 @@ def __init__(
     def native_value(self) -> float | datetime | None:
         """Return the state of the sensor."""
         return self.entity_description.value_fn(self.coordinator.data)
+
+    @property
+    def extra_state_attributes(self) -> dict[str, None | str | dict[datetime, float]]:
+        """Return the state attributes."""
+        if self.entity_description.key != "average_price":
+            return None
+
+        data = self.coordinator.data
+
+        return {
+            "energy": data.energy.prices,
+            "energy_template": self.coordinator.energy_modifyer.template,
+            "gas": data.gas_today.prices if data.gas_today is not None else None,
+            "gas_template": self.coordinator.gas_modifyer.template,
+        }
```

Comment:
@klaasnicolaas I know this has been originally removed from the integration: https://github.com/home-assistant/core/pull/83886#discussion_r1059137921

Do you know if there is any way we can add this so it would be approved? Maybe by adding a new sensor that has the price in a JSON encoded string?